tests/unit/type/list.tcl

```diff
@@ -575,45 +668,90 @@ start_server {
             set rd [redis_deferring_client]
             r del blist1{t} blist2{t}
             r set blist2{t} nolist{t}
+            bpop_command_two_key $rd $pop blist1{t} blist2{t} 1
             $rd $pop blist1{t} blist2{t} 1
```

Comment:
hi @enjoy-binbin I think this should be deleted, no?
can you please PR?

Reply:
opps, it is actually the line below it that needs to be deleted
`$rd $pop blist1{t} blist2{t} 1`